DataFormats/SiStripDetId/interface/SiStripDetId.h

```diff
@@ -112,18 +112,18 @@ class SiStripDetId : public DetId {
 // ---------- inline methods ----------
```

Comment:
@ngrenz , yes, it is the most effective c++ but I agree the readability is not the best. It is possible to have something like:

inline uit32_t SiStripDetId::stereo const
{
   uit32_t k = 0;
   if(1 == ((id_>>sterStartBit_ ) & sterMask_ )) { k = 1; }
   return k;
}
similar solutions may be applied in other inlined methods from this header.